index.html

```diff
@@ -1 +1,39 @@
-task1-3
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+    <meta charset="utf-8">
+    <title>task3</title>
+</head>
+<body>
+    <script>
+        
+        
+        
+        function task (){  
+            console.log("----------------------");
+            console.log("----------------------");
+            console.log("現在持っているタスク一覧");
+            console.log("----------------------");
+            console.log("----------------------");
+
+            for(let i = 0; i < array.length; i++){
+            console.log(i + ":" + array[i]);
+            }
+        }
+
+        array= ["掃除","買い物","散歩"];
+        task();
+
+        let result = window.prompt("タスクを入力してください");
```

Comment:
```suggestion
        const result = window.prompt("タスクを入力してください");
```

再代入はしないと思うので `const` でよいと思います